styles/style.css

```diff
@@ -0,0 +1,37 @@
+header {
+    background-color:rgb(31, 113, 101);
+}
+body{
+    font-family:Georgia, 'Times New Roman', Times, serif
+}
+#main {
+    background-image: url('https://thumbs.gfycat.com/QuestionablePeriodicGharial-size_restricted.gif');
```

Comment:
This image really is the content of the section it appears in. Rather than using a background-image rule, prefer using an actual img tag.